srcpkgs/python3-trimesh/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'python3-trimesh'
 pkgname=python3-trimesh
 version=3.6.43
-revision=1
+revision=2
```

Comment:
I don't think we need to rev bump.
I maybe wrong.
Let me try out when this one is ready ;)

Reply:
I did one better and updated to the latest version, so the revbump is not relevant anymore.